Dockerfile.clr.boringssl.envoy

```diff
@@ -0,0 +1,39 @@
+FROM clearlinux:base as builder
+
+env HOME /root
+RUN rm -rf /run/lock/clrtrust.lock
+RUN clrtrust generate
+RUN swupd update || swupd update
+RUN swupd bundle-add os-core-dev python2-basic kvm-host
+
+RUN curl -LO https://github.com/bazelbuild/bazel/releases/download/1.1.0/bazel-1.1.0-installer-linux-x86_64.sh
+RUN chmod +x bazel-1.1.0-installer-linux-x86_64.sh
+RUN ./bazel-1.1.0-installer-linux-x86_64.sh --user
+
+RUN git clone --recurse-submodules --single-branch --branch qat-provider-master https://github.com/ipuustin/envoy.git envoy
+
+# build Envoy
+RUN cd envoy && /root/.bazel/bin/bazel build --verbose_failures -c opt -s //source/exe:envoy-static
+
+# Start creating the target image
+FROM clearlinux:base
```

Comment:
have you checked the way we build the minimal device plugins images using `swupd os-install`.  `libstdc++` has a dedicated bundle: `libstdcpp`.

Reply:
Ok, will fix.